src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx

```diff
@@ -162,6 +162,34 @@ describe("when pressing bid button", () => {
           })
         )
       })
+
+      it("displays an error message on a mutation failure", () => {
+        const error = {
+          message:
+            'https://stagingapi.artsy.net/api/v1/bidder?sale_id=leclere-impressionist-and-modern-art - {"error":"Invalid Sale"}',
```

Comment:
A little confusing-- the url in this error message is from a `createBidder` mutation it looks like, not a `createBidderPosition` mutation (although they'll likely have the same error message, so this is #minor).

Reply:
Right, I had the same feeling. I'll update this to a more realistic error message.